plugin/kubernetes/README.md

```diff
@@ -225,3 +225,17 @@ or the word "any"), then that label will match all values.  The labels that acce
 *.service.default.svc.cluster.local. 5	IN A	192.168.25.15
 ```
  This response can be randomized using the `loadbalance` plugin
+
+## Metadata
+
+The kubernetes plugin will publish the following metadata, if the metadata
```

Comment:
```suggestion
The kubernetes plugin will publish the following metadata, if the _metadata_
```

I think most of the time we _italicize_ plugin names when referring to the plugin.  So here, first metadata not italicized, second one italicized. Admittedly, we are not 100% consistent throughout all the READMEs (case in point `loadbalance` in the section above should be _loadbalance_).